README.md

```diff
@@ -21,6 +21,20 @@ add_action( 'init', 'opt_my_cpt_into_autoshare' );
 
 In addition, adding support while registering custom post types also works. Post types are automatically set to autoshare. Future versions of this plugin could allow this to be set manually.
 
+While the autoshare feature can be opted into for post types using the above filter, by default the editor still has to manually enable autoshare during the post prepublish flow. The `autoshare_for_twitter_enabled_default` filter allows autoshare to be enabled by default for all posts of a given post type. Editors can still manually uncheck the option during the publishing flow. 
+
+Example:
+
+```php
+function enable_autoshare_by_default_for_core_post_type( $enabled, $post_type ) {
+	if ( 'post' === $post_type ) {
+		return true;
+	}
+
+	return $enabled;
+}
+add_filter( 'autoshare_for_twitter_enabled_default' 'enable_autoshare_by_default_for_core_post_type', 10, 2 );
```

Comment:
@johnwatkins0 Missing comma between filter and function name.

Reply:
Thanks! Updated.